app/evm_keeper.go

```diff
@@ -0,0 +1,85 @@
+package app
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	bankkeeper "github.com/cosmos/cosmos-sdk/x/bank/keeper"
+	evmtypes "github.com/tharsis/ethermint/x/evm/types"
+)
+
+// Convertion between native gas KAVA (6) to EVM (18).
+// 12 decimal difference, so 1_000_000_000_000.
+var conversionMultiplier = sdk.NewInt(1_000_000_000_000)
+
+// EVMBankKeeper is a wrapper for bank keeper that converts between EVM and native decimals
+type EVMBankKeeper struct {
+	bankKeeper bankkeeper.Keeper
+}
+
+var _ evmtypes.BankKeeper = (*EVMBankKeeper)(nil)
+
+// NewEVMBankKeeper returns a wrapped bank keeper that converts between EVM and native decimals
+func NewEVMBankKeeper(bk bankkeeper.Keeper) EVMBankKeeper {
```

Comment:
It'd be good to unit test all of the functions here -- table tests with specific examples would be great for a wrapper like this.